sphinx/writers/latex.py

```diff
@@ -586,13 +589,10 @@ def __init__(self, document, builder):
             logger.warning(__('no Babel option known for language %r'),
                            builder.config.language)
 
-        # simply use babel.get_language() always, as get_language() returns
-        # 'english' even if language is invalid or empty
-        self.elements['classoptions'] += ',' + self.babel.get_language()
```

Comment:
I'm moving this to both `babel` and `polyglossia` cases.
After this refactoring, all of `babel`, `polyglossia` and `multilingual` key become empty string for `platex` engine. So this line will not be called for the engine.